### DISCUSSION TITLE
how to add schemas in burn-my-windows

### DISCUSSION THREAD

**Discussion by daveheart73** [Schneegans/Burn-My-Windows#391] [Q&A] [answered]
Hello
I woud to create a schema to activate incinerate effect instead the fire default
This is my script 

gsettings --schemadir ~/.local/share/gnome-shell/extensions/burn-my-windows@schneegans.github.com/schemas set org.gnome.shell.extensions.burn-my-windows active-profile incinerate-effect="true"

At launch there's no errors but there's no effect 
Where i wrong?
Thanks 
Dave

**Comment by Schneegans** [comment, marked as answer]
The effect profile configurations are stored in `~/.config/burn-my-windows/profiles`. If you have only one profile, there will be one file in this directory (with a random name). You want to have at least these lines in there:

```
[burn-my-windows-profile]
fire-enable-effect=false
incinerate-enable-effect=true
```

Does this help?

**Comment by daveheart73** [comment]
Thank You so much
So ... it's possible to change effect directly modifiyng this file instead using gsettings --schemas?
Dave

**Reply by Schneegans** [reply]
Yes. In fact, the changes you make to the file are immediately applied. You can even watch the settings widgets in the extension's preferences dialog change when you save the file :wink:

**Comment by daveheart73** [comment]
Super
Thank you so much
Dave

Il mar 24 ott 2023, 11:30 Simon Schneegans ***@***.***> ha
scritto:
